ui-tests/src/test/java/io/syndesis/qe/pages/WizardPageObject.java

```diff
@@ -0,0 +1,42 @@
+package io.syndesis.qe.pages;
+
+import static java.util.Arrays.asList;
+
+import java.util.List;
+import java.util.ListIterator;
+
+import io.syndesis.qe.pages.interfaces.wizard.WizardStep;
+
+public abstract class WizardPageObject extends SyndesisPageObject {
+
+	private List<WizardStep> wizardSteps = null;
+	private ListIterator<WizardStep> steps = null;
+	private SyndesisPageObject currentStep = null;
+
+ public void setSteps (List<WizardStep> stepPages) {
+	 wizardSteps = stepPages;
+	 initIteration();
+ }
+
+ public void setSteps (WizardStep [] stepPages) {
+	 wizardSteps = asList(stepPages);
+	 initIteration();
+ }
+
+ private void initIteration() {
+	 steps = wizardSteps.listIterator();
+	 steps.next();
+	 currentStep = (SyndesisPageObject) wizardSteps.get(0);
+ }
```

Comment:
Formating issues

Reply:
I can't see any

Reply:
setSteps and initIteration methods don't have indentation.

Reply:
thanks. fixed